### ISSUE TITLE
Allow dependencies to be ignored when uninstalling a package

### ISSUE THREAD

**Issue by DarwinJS** [chocolatey/choco#892]
Scenario:
I install visual studio test agents.

It depends on VirtualCloneDrive - only for the the install to mount an ISO.

VirtualCloneDrive leaves a shortcut on the desktop and I'd like to remove any excess virtual drivers that aren't necessarily tested in cloud environments.

Doing "choco uninstall" gives a message that it cannot be removed because something depends on it.

In this case I know the dependency was only for installation, so I would like to remove it anyway.

Let me know if there is a work around.  However, even if there is a work around -ignoredependencies might be easier to find and a little more consistent with "choco install" ?

**Comment by ferventcoder** [comment]
Agreed and this is something we want to do. Unfortunately NuGet.Core doesn't offer this ability currently, so this is not easily achievable. But we definitely agree that this is a sought after feature.
